blivetgui/visualization/logical_view.py

```diff
@@ -246,22 +298,52 @@ def _get_child_position(self, child_iter):
             return "inner"
 
     def select_rectanlge(self, device):
+        """
+        Select all rectanlgelge.
+
+        Args:
+            self: (todo): write your description
+            device: (todo): write your description
+        """
         for rect in self.rectangles:
             if rect.device == device:
                 rect.set_active(True)
 
     def _on_rectangle_toggle(self, button):
+        """
+        Toggle the given button.
+
+        Args:
+            self: (todo): write your description
+            button: (todo): write your description
+        """
         if not self._ignore_toggle:
             self._ignore_toggle = True
             self.blivet_gui.list_partitions.select_device(button.device)
             self._ignore_toggle = False
 
     def _on_button_release(self, button, event):
+        """
+        Handle keyboard release events.
+
+        Args:
+            self: (todo): write your description
+            button: (todo): write your description
+            event: (todo): write your description
+        """
         if event.button == 3:
             self._on_rectangle_toggle(button)  # select the button
             self.blivet_gui.popup_menu.menu.popup_at_pointer(None)
 
     def _on_button_press(self, button, event):
+        """
+        Button press press press press.
```

Comment:
Artificial intelligence won't take over the world, but artificial stupidity might.